sale_order_price_recalculation/models/sale_order.py

```diff
@@ -15,3 +19,11 @@ def action_update_names(self):
         lines = self.mapped("order_line")
         lines._compute_name()
         return True
+
+    def _recalculate_prices(self):
+        self = self.browse(self._context.get("active_ids"))
```

Comment:
@bealdav _context is deprecated since long time ago : https://github.com/odoo/odoo/blob/16.0/odoo/models.py#L5160

Reply:
Yes deprecated since 10.0, 11.0 ? but yet here, arf !
Yes I can remove